### PR TITLE
Updated the dashboard documentation

### PR DESCRIPTION
**Pull request by gertipoppel**
1. I updated the screenshots once again
2. Added that system applications may be running (therefore the zero state page has changed again)
3. Removed the section on "accessing the dashboard"
4. Updated the section on "Observe the Kubernetes Graphical dashboard (optional) in the Hello World Walkthrough
5. The Dashboard tour (Web interface) is now accessible from 
a) Guides --> Getting Started --> Hello World Walkthrough
and
b) Application Management and Deployment 

One question: how are titles added to pages? I have removed the section on accessing the dashboard and created a new page ui-access.md. Not sure whether this is still needed at all. 

This PR addresses the following issue: https://github.com/kubernetes/dashboard/issues/538

@johndmulhausen @bryk Please review

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1,2 +0,0 @@
-kubernetes.io
```
Comment (johndmulhausen) [inline CNAME]: Back out any changes to this file

File: docs/user-guide/ui-access.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (johndmulhausen) [inline docs/user-guide/ui-access.md]: Nice! But this file needs a spot in the left nav. Can you add one in guides.yml?

**Comment by bryk** [comment]
LGTM when comments from @johndmulhausen are resolved :)

**Comment by bryk** [comment]
@gertipoppel You should squash your changes into one change. If needed, ask some of your local engineers to help you or refer to online help like [0] [1]

[0] https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/
[1] http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by gertipoppel** [comment]
@johndmulhausen PTAL

File: docs/hellonode.md [hunk 3]
```diff
@@ -254,26 +254,20 @@ While this is happening, the users of the services should not see any interrupti
 
 Hopefully with these deployment, scaling and update features you’ll agree that once you’ve setup your environment (your GKE/Kubernetes cluster here), Kubernetes is here to help you focus on the application rather than the infrastructure.
 
-## Observe the Kubernetes Graphical dashboard (optional)
+## Observe the Kubernetes web user interface (optional)
 
-While logged into your development machine, execute the following commands:
+With Kubernetes 1.2, a graphical web user interface (dashboard) has been introduced. It is enabled by default for 1.2 clusters.
+This user interface allows you to get started quickly and enables the same functionality found in the CLI as a more approachable and discoverable way of interacting with the system.
```
Comment (cheld) [inline docs/hellonode.md]: > same functionality found

what about similar functionality (as it is just a small subset)?

**Comment by cheld** [comment]
LGTM

File: _data/guides.yml [hunk 4]
```diff
@@ -16,6 +16,8 @@ toc:
     path: /docs/user-guide/walkthrough/
   - title: Kubernetes 201
     path: /docs/user-guide/walkthrough/k8s201/
+  - title: Accessing the Kubernetes web user interface
```
Comment (johndmulhausen) [inline _data/guides.yml]: How about: Accessing the Kubernetes Web UI

**Comment by johndmulhausen** [comment]
Got a nit with the proposed title; at a minimum, it's gotta have Title casing. Almost there!

**Comment by johndmulhausen** [comment]
Side note: Do any of those dashboard images re-use images that you merged in your previous pull request? If so, re-use those and drop them out of this PR

Also, looks like you have a merge conflict now that must be resolved. :(

**Comment by johndmulhausen** [comment]
Ping @gertipoppel

**Comment by gertipoppel** [comment]
@johndmulhausen PTAL
Sorry for the late reply, I was not in the office for 10 days...
I have included the comments by you and cheld, and hope that this PR can be merged:-)

**Comment by bryk** [comment]
@gertipoppel LGTM again :) See my previous comment about squashing "You should squash ..."

**Comment by gertipoppel** [comment]
@bryk Done. PTAL
Sorry for this.

**Comment by gertipoppel** [comment]
@johndmulhausen PTAL
Do I need to do anything else so that this gets merged?

**Comment by gertipoppel** [comment]
@johndmulhausen Could you please merge - I would like to create a new fork and currently my master is messed up. Thank you. I will then include additional updates to the dashboard tour.

**Comment by roberthbailey** [comment]
@bgrant0607 @pwittrock Can we please merge this PR?

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**